apps/broadcast-app-indexer/drizzle/0000_init.sql

```diff
@@ -0,0 +1,66 @@
+CREATE SCHEMA "broadcast_app_indexer_offchain";
+--> statement-breakpoint
+CREATE EXTENSION IF NOT EXISTS "pgcrypto";
+--> statement-breakpoint
+CREATE TYPE "broadcast_app_indexer_offchain"."neynar_notification_service_queue_status" AS ENUM('pending', 'processing', 'completed', 'failed');--> statement-breakpoint
+CREATE TABLE "broadcast_app_indexer_offchain"."auth_siwe_refresh_token" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"session_id" uuid NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"is_used" boolean DEFAULT false NOT NULL,
+	"expires_at" timestamp with time zone NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE "broadcast_app_indexer_offchain"."auth_siwe_session" (
+	"id" uuid PRIMARY KEY DEFAULT gen_random_uuid() NOT NULL,
+	"created_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"updated_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"last_used_at" timestamp with time zone DEFAULT now() NOT NULL,
+	"user_id" text NOT NULL
+);
+--> statement-breakpoint
+CREATE TABLE "broadcast_app_indexer_offchain"."channel_subscription_farcaster_notification_settings" (
+	"channel_id" numeric(78, 0) NOT NULL,
+	"app_id" text NOT NULL,
+	"client_fid" integer NOT NULL,
+	"user_address" text NOT NULL,
+	"user_fid" integer NOT NULL,
```

Comment:
not sure that Base Mini-apps will always have this user identifier, best to check

Reply:
the fid is used only for farcaster

Reply:
https://docs.base.org/mini-apps/features/context#user-object they do have `fid`